### PR TITLE
var is not really a type

### PR DESCRIPTION
**Pull request by andreycha**
- use quotation mark around "type"

Strictly speaking `var` is not a type, it's a syntactic sugar. ["Implicitly typed local variables"](https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/classes-and-structs/implicitly-typed-local-variables.md) article use quotation marks to point that out, which is correct.

### CODE REVIEW

File: docs/csharp/language-reference/keywords/var.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ translation.priority.ht:
   - "zh-tw"
 ---
 # var (C# Reference)
-Beginning in Visual C# 3.0, variables that are declared at method scope can have an implicit type `var`. An implicitly typed local variable is strongly typed just as if you had declared the type yourself, but the compiler determines the type. The following two declarations of `i` are functionally equivalent:  
+Beginning in Visual C# 3.0, variables that are declared at method scope can have an implicit "type" `var`. An implicitly typed local variable is strongly typed just as if you had declared the type yourself, but the compiler determines the type. The following two declarations of `i` are functionally equivalent:
```
Comment (BillWagner) [inline docs/csharp/language-reference/keywords/var.md]: How about this wording instead:

> ... variables that are declared at method scope can be declared using the `var` keyword, indicating that it is an implicitly typed local variable.

You are correct that calling `var` a type is wrong. I'd like to avoid suggesting that abstraction.

**Review by BillWagner — CHANGES_REQUESTED** [review]
@andreycha Thanks for spotting this and suggesting a fix. We really appreciate it. I agree that as written, the text is incorrect.

What do you think of my suggested revision? I think it makes the description more clear.

File: docs/csharp/language-reference/keywords/var.md (hunk 1 above)
Comment (andreycha) [inline docs/csharp/language-reference/keywords/var.md]: Agree. Should we get rid of mentioning the same abstraction in ["Implicitly typed local variables"](https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/classes-and-structs/implicitly-typed-local-variables.md) article? The very first sentences say:

> Local variables can be given an inferred "type" of `var` instead of an explicit type. The `var` keyword instructs the compiler to infer the type of the variable from the expression on the right side of the initialization statement.

We can change it something like:

> Local variables can be declared without giving an explicit type. The `var` keyword instructs the compiler to infer the type of the variable from the expression on the right side of the initialization statement.

Comment (BillWagner) [inline docs/csharp/language-reference/keywords/var.md]: Yes, that's a great suggestion. Do you want to add it to this PR?

Comment (andreycha) [inline docs/csharp/language-reference/keywords/var.md]: Sure. Done.

**Review by BillWagner — APPROVED** [review]

File: docs/csharp/language-reference/keywords/var.md (hunk 1 above)
Comment (BillWagner) [inline docs/csharp/language-reference/keywords/var.md]: Thanks for making the update.  I'll merge this as soon as the build finishes.

Comment (andreycha) [inline docs/csharp/language-reference/keywords/var.md]: Thank you!
